### PR TITLE
Refactor packages location

### PR DESCRIPTION
**Pull request by aleskandro**
~~Blocked/Stacked by #10 (needs rebase after merging #10)~~, this PR:
- Changes the name of the module to reflect the github repository
- Consolidates the imports ordering (golang-native, some 3rd parties, k8s, ocp, other 3rd parties, this operator's packages)
- Refactor packages location to simplify folders structure and the ginkgo suites discovery mechanism for integration tests

### CODE REVIEW

**Comment by Prashanth684** [comment]
how did the security issues get fixed? i still see that seed.go has the same code

**Comment by aleskandro** [comment]
from the webui in snyk

File: controllers/operator/podplacementconfig_controller.go [hunk 1]
```diff
@@ -14,19 +14,20 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
```
Comment (Prashanth684) [inline controllers/operator/podplacementconfig_controller.go:16]: it's weird that inside the controller there is an operator repo with the main controllers...shouldn't this be outside ?

**Pull request merged**